source/collimation.f90

```diff
@@ -3030,14 +2785,14 @@ subroutine collimate_do_collimator(stracki)
 ! --- TW JUNE08
           if (firstrun) then
             write(collsettings_unit,'(a,1x,i10,5(1x,e13.5),1x,a)')      &
-     &                       db_name1(icoll)(1:12),                     &
+     &                       cdb_cNameUC(icoll)(1:12),                     &
```

Comment:
Are collimator names strictly limited to 12 characters long?

Reply:
No, not at all. We need to go through all the file outputs at some point. This line was edited by a search/replace, so not touched manually. But there are a few of these around.